src/processing.jl

```diff
@@ -436,10 +439,13 @@ function modelfit(data::RheoTimeData,
                   lo::Union{NamedTuple,Nothing,Dict} = nothing,
                   hi::Union{NamedTuple,Nothing,Dict} = nothing,
                   verbose::Bool = false,
-                  rel_tol = 1e-4,
+                  rel_tol::Union{Real,Nothing} = 1e-4,
+                  rel_tol_f::Union{Real,Nothing} = nothing, 
```

Comment:
```
rel_tol_f::Union{Real,Nothing} = nothing,
rel_tol::Union{Real,Nothing} = isnothing(rel_tol_f) ? 1e-4 : nothing,
```

Reply:
This is now the standard for all the functions where these stopping criteria are set. All stopping criteria are optional, and if none are specified, a relative input change criterion (x_tol) is set. 

The behaviour is such that if the user specifies a stopping criterion, it will be applied. If an f_tol limit is supplied, then it will override the default x_tol setting, and only the f_tol value will be used to set a limit. If both x_tol and f_tol limits are supplied, they will both be applied*.

*N.b. this can lead to unpredictable behaviour as it is not clear which stopping criteria will apply in any particular circumstance, and it is likely to depend on the starting point of the optimisation, as well as the objective function landscape.